aten/src/ATen/native/native_functions.yaml

```diff
@@ -596,3 +596,57 @@
 
 - func: _cudnn_init_dropout_state(Type ty, double dropout, bool train, int64_t dropout_seed) -> Tensor
   variants: function
+
+# Unary Ops #
+
+- func: ceil(Tensor self) -> Tensor
```

Comment:
Stick to the convention in the file. Sort functions alphabetically.

Reply:
Will do at the end.